### DISCUSSION TITLE
CROSSHAIR for Orientation in a large mind map

### DISCUSSION THREAD

**Discussion by Engesser** [freeplane/freeplane#675] [Ideas]
Orientation in a large mind map 

If you want to move quickly in a very large mindmap you can use the excellent overview map.
The problem is: you can only guess where you are, even if you know that the node you are looking for is "about there"! 
In my opinion it would be very interesting if a crosshair would run with the mouse and its coordinates at the edge would run continuously as x,y-coordinates. Then one could briefly note a pair of numbers and could quickly and accurately "drive around" in the map.

Would be nice if the proposal would find support?

Greetings, Karl

**Comment by wordmuse** [comment]
"You are here" is always helpful with a large map.
